L1Trigger/L1CaloTrigger/test/ntuples/L1TCaloTriggerNtupleBase.h

```diff
@@ -0,0 +1,14 @@
+#include "L1Trigger/L1THGCalUtilities/interface/HGCalTriggerNtupleBase.h"
+
+class L1TCaloTriggerNtupleBase : public HGCalTriggerNtupleBase {
+public:
+  L1TCaloTriggerNtupleBase(const edm::ParameterSet& conf)
+      : HGCalTriggerNtupleBase(conf),
+        branch_name_prefix_(conf.getUntrackedParameter<std::string>("BranchNamePrefix", "")) {}
+  ~L1TCaloTriggerNtupleBase() override{};
+
+  char const* branch_name_w_prefix(const std::string name) const { return (branch_name_prefix_ + "_" + name).c_str(); }
```

Comment:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-08-25-1100/src/L1Trigger/L1CaloTrigger/test/ntuples/L1TCaloTriggerNtupleBase.h:10:75: warning: returning address of local temporary object [-Wreturn-stack-address]